props/saber_chris_buttons.h

```diff
@@ -0,0 +1,1036 @@
+    if (!mode_volume_) {
+      On();
+    } else {
+     if (fusor.angle1() > 0) {
+      VolumeUp();
+    } else {
+      VolumeDown();
+    }
+    }
+    return true;
+
+
+// Modified 2 and 3 button 'next/previous preset' AND volume down,
+// to align with multiple skips above.
+// Forward one font pointing up, back one font pointing down.
+  case EVENTID(BUTTON_AUX, EVENT_FIRST_SAVED_CLICK_SHORT, MODE_OFF):
+    // backwards if pointing down
+    if (!mode_volume_) {
+    SetPreset(current_preset_.preset_num + (fusor.angle1() < -M_PI / 4 ? -1 : 1), true);
+        } else {
+      VolumeDown();
+    }
+#ifdef SAVE_PRESET
+    SaveState(current_preset_.preset_num);
+#endif
+    return true;
+
+
+// 1 button 'next/previous preset' command.
+// Forward one font pointing up, back one font pointing down.
+#if NUM_BUTTONS == 1 
+      case EVENTID(BUTTON_POWER, EVENT_SECOND_HELD, MODE_OFF):
+    // backwards if pointing down
+    if (!mode_volume_) {
+    SetPreset(current_preset_.preset_num + (fusor.angle1() < -M_PI / 4 ? -1 : 1), true);
+        } else {
+      VolumeDown();
+    }
+#ifdef SAVE_PRESET
+    SaveState(current_preset_.preset_num);
+#endif
+    return true;
+#endif
+
+
+//  1 Button skips to first preset (up) or last preset (down)
+//  or middle preset if horizontal:
+#if NUM_BUTTONS == 2
+case EVENTID(BUTTON_AUX, EVENT_FIRST_HELD_LONG, MODE_OFF):
+#endif
+case EVENTID(BUTTON_POWER, EVENT_FIRST_HELD_LONG, MODE_OFF):
+#define DEGREES_TO_RADIANS (M_PI / 180)
+ if (fusor.angle1() > 45 * DEGREES_TO_RADIANS) {
+// If pointing up
+SetPreset(0, true);
+ } else if (fusor.angle1() < -45 * DEGREES_TO_RADIANS) {
+// If pointing down
+SetPreset(-1, true);
+ } else {
+// If horizontal
+  CurrentPreset tmp;
+  tmp.SetPreset(-1);
+  SetPreset(tmp.preset_num / 2, true);
+}
+#ifdef SAVE_PRESET
+    SaveState(current_preset_.preset_num);
+#endif
+    return true;
+
+
+#if NUM_BUTTONS == 2
+// 1 and 2 button: Previous Preset, retained legacy control.
+    case EVENTID(BUTTON_POWER, EVENT_CLICK_SHORT, MODE_OFF | BUTTON_AUX):
+#endif
+    if (!mode_volume_) {
+      previous_preset();
+    }
+    return true;
+
+
+//  Skips to first preset (up) or last (battery or charge) preset (down)
+//  or middle preset if horizontal:
+#if NUM_BUTTONS == 2
+// 2 button: First Preset
+case EVENTID(BUTTON_AUX, EVENT_HELD_LONG, MODE_OFF):
+#endif
+#define DEGREES_TO_RADIANS (M_PI / 180)
+ if (fusor.angle1() > 45 * DEGREES_TO_RADIANS) {
+// If pointing up
+SetPreset(0, true);
+ } else if (fusor.angle1() < -45 * DEGREES_TO_RADIANS) {
+// If pointing down
+SetPreset(-1, true);
+ } else {
+// If horizontal
+  CurrentPreset tmp;
+  tmp.SetPreset(-1);
+  SetPreset(tmp.preset_num / 2, true);
+}
+#ifdef SAVE_PRESET
+    SaveState(current_preset_.preset_num);
+#endif
+    return true;
+    
+
+//  Manual Blade ID Options
+case EVENTID(BUTTON_POWER, EVENT_THIRD_SAVED_CLICK_SHORT, MODE_OFF):
+#ifdef FAKE_BLADE_ID
+//  Toggles between blade arrays regardless of BladeID status.
```

Comment:
Comments should be indented too, not starting at the beginning of the line (unless they are a comment outside of all classes and functions.)

Reply:
Again, hopefully fixed now. :)